README.md

```diff
@@ -5,6 +5,7 @@
 [![CRAN_Downloads_Badge](https://cranlogs.r-pkg.org/badges/grand-total/DGEobj?color=9bc2cf)](https://cran.r-project.org/package=DGEobj) 
 [![Travis build status](https://travis-ci.com/cb4ds/DGEobj.svg?branch=master)](https://travis-ci.com/cb4ds/DGEobj?branch=master)
```

Comment:
Remove the travis build status and put the CircleCI status in this location (before code coverage)